src/components/MyCommunity/index.tsx

```diff
@@ -62,9 +60,12 @@ export default function MyCommunity() {
     }
     return 'Display name';
   };
-  useEffect(() => {
-    queryCommunities();
-  }, []);
+
+  useFocusEffect(
+    useCallback(() => {
```

Comment:
do we need `useCallback`

Reply:
Yes. 
useFocusEffect does not have dep array and we use useCallback dep array instead.